core/services/pipeline/task.bridge.go

```diff
@@ -28,6 +28,10 @@ func (t *BridgeTask) Type() TaskType {
 	return TaskTypeBridge
 }
 
+func (t *BridgeTask) ApplyDefaults(inputValues map[string]string, g TaskDAG, self taskDAGNode) error {
```

Comment:
If we define this return nil on the BaseTask we don't have to define it on every task, just override where needed

Reply:
spoke offline - want to make it explicit